README

```diff
@@ -293,6 +293,11 @@ Features
       collectd without the need to start a heavy interpreter every interval.
       See collectd-python(5) for details.
 
+    - radio
+      The radio plugin records whether WiFi, Bluetooth and other radios are
+      switched on or off. For that, it uses rfkill interface exposed by Linux
```

Comment:
Just to be sure that I haven't missed anything, I am adding one-per-one reply

* I specified that its rfkill sysfs interface (or do you want full name from https://www.kernel.org/doc/Documentation/ABI/stable/sysfs-class-rfkill ?)

* sentence ended with the dot